tests/integration/test_project.py

```diff
@@ -143,3 +143,21 @@ def test_many_indexes(PipenvInstance, pypi):
             f.write(contents)
         c = p.pipenv('install')
         assert c.return_code == 0
+
+
+@pytest.mark.install
+@pytest.mark.project
+def test_rewrite_outline_table(PipenvInstance, pypi):
+    with PipenvInstance(pypi=pypi, chdir=True) as p:
+        with open(p.pipfile_path, 'w') as f:
+            contents = """
+[packages.requests]
+version = "*"
+            """.strip()
+            f.write(contents)
```

Comment:
Not entirely related to this PR, but it just occurred to me that we need a test to make sure we are reading Pipfile and Pipfile.lock with the correct encoding. It is likely best to have an abstraction around this in `Project` too (something like `with project.open_pipfile('w') as f:`).

Reply:
We had this in `Project` -- did it go away?